pkg/apis/core/v1beta1/types.go

```diff
@@ -63,6 +63,7 @@ type DataVolumeSource struct {
 	Upload   *DataVolumeSourceUpload   `json:"upload,omitempty"`
 	Blank    *DataVolumeBlankImage     `json:"blank,omitempty"`
 	Imageio  *DataVolumeSourceImageIO  `json:"imageio,omitempty"`
+	VDDK     *DataVolumeSourceVDDK     `json:"vddk,omitempty"`
```

Comment:
Please update the alpha API as well.  We committed to supporting alpha for a little longer.